### PR TITLE
Assets non-relative path with timestamp

### PR DESCRIPTION
**Pull request by bizley**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #18529

### CODE REVIEW

File: framework/web/AssetManager.php [hunk 1]
```diff
@@ -316,14 +316,20 @@ protected function loadDummyBundle($name)
      * The actual URL is obtained by prepending either [[AssetBundle::$baseUrl]] or [[AssetManager::$baseUrl]] to the given asset path.
      * @param AssetBundle $bundle the asset bundle which the asset file belongs to
      * @param string $asset the asset path. This should be one of the assets listed in [[AssetBundle::$js]] or [[AssetBundle::$css]].
+     * @param bool|null $overrideAppendTimestamp Whether to override manager's appendTimestamp property
      * @return string the actual URL for the specified asset.
      */
-    public function getAssetUrl($bundle, $asset)
+    public function getAssetUrl($bundle, $asset, $overrideAppendTimestamp = null)
```
Comment (samdark) [inline framework/web/AssetManager.php:322]:
```suggestion
    public function getAssetUrl($bundle, $asset, $appendTimestamp = null)
```

Comment (samdark) [reply framework/web/AssetManager.php:322]: I don't think "override" makes sense from end user standpoint.

**Review by samdark — APPROVED** [review]

File: framework/web/AssetManager.php [hunk 2]
```diff
@@ -316,14 +316,20 @@ protected function loadDummyBundle($name)
      * The actual URL is obtained by prepending either [[AssetBundle::$baseUrl]] or [[AssetManager::$baseUrl]] to the given asset path.
      * @param AssetBundle $bundle the asset bundle which the asset file belongs to
      * @param string $asset the asset path. This should be one of the assets listed in [[AssetBundle::$js]] or [[AssetBundle::$css]].
+     * @param bool|null $appendTimestamp Whether to override manager's appendTimestamp property
```
Comment (samdark) [inline framework/web/AssetManager.php:319]:
```suggestion
     * @param bool|null $appendTimestamp Whether to append timestamp to the URL.
```

Comment (bizley) [reply framework/web/AssetManager.php]: Well, if it's null, the manager's value takes over so that change might be confusing

**Comment by samdark** [comment]
👍

File: framework/web/AssetManager.php (hunk 2 above)
Comment (bizley) [reply framework/web/AssetManager.php]: hmm, ok then :P

**Comment by brandonkelly** [comment]
Thanks @bizley!

**Pull request merged**